src/css/doc.css

```diff
@@ -135,6 +135,7 @@
 
 .doc .tableblock p {
   font-size: inherit;
+  line-break: anywhere;
```

Comment:
Doesn't this split words at any point, even if they shouldn't be split from a grammatical perspective?